layouts/partials/footer.hbs

```diff
@@ -15,13 +15,31 @@
         </ul>
       </div>
 
-      <p>© Node.js Foundation. All Rights Reserved. Portions of this site originally © Joyent.</p>
-      <p>Node.js is a trademark of Joyent, Inc. and is used with its permission. Please review the <a href="/static/documents/trademark-policy.pdf">Trademark Guidelines of the Node.js Foundation</a>.</p>
-      <p>Linux Foundation is a registered trademark of The Linux Foundation.</p>
-      <p>Linux is a registered <a href="http://www.linuxfoundation.org/programs/legal/trademark" title="Linux Mark Institute">trademark</a> of Linus Torvalds.</p>
-      <p>
-        <a href="https://raw.githubusercontent.com/nodejs/node/master/LICENSE">Node.js Project Licensing Information</a>.
-      </p>
+      <div class="linuxfoundation-content">
+        <div>
```

Comment:
Extra `div`

Reply:
It's for the display flex (2 childs)

Reply:
Well, divs without classes is bad IMO. Also see my comment about style.css being used by other nodejs sites, which means you can't do such changes.

Reply:
Mmh ok, what can I do ?

Reply:
Not add new elements, or wait until we solve the CSS issue in the other repos.

Reply:
I mean, it will be hard to add the contributor card without adding elements, I guess I will wait :)

Reply:
Sure, but this should be ES5 and use elements with proper classes and everything. Empty `div`s is pretty bad.

Reply:
You are right